<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(238, 228, 155);
border-radius:20px</string>
  </property>
  <widget class="QWidget" name="triosWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedLights">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="select">
       <widget class="QListWidget" name="selectLights">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>231</width>
          <height>691</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Monaco</family>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0.0397727, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
alternate-background-color: rgb(74, 78, 81);

color: rgb(123, 22, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerItem</enum>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::ListMode</enum>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
        <property name="selectionRectVisible">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>151</width>
          <height>141</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Monaco</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:20px;
selection-background-color: rgb(255, 13, 13);
font: 14pt &quot;Monaco&quot;;
selection-color: rgb(255, 12, 31);
color: rgb(123, 22, 255);</string>
        </property>
        <property name="text">
         <string>Off</string>
        </property>
       </widget>
       <widget class="QPushButton" name="statusButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>180</y>
          <width>151</width>
          <height>141</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Monaco</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:20px;
font: 14pt &quot;Monaco&quot;;
color: rgb(123, 22, 255);</string>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>21</width>
          <height>691</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="light">
       <widget class="QLCDNumber" name="lcdValue">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>10</y>
          <width>201</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(123, 22, 255);</string>
        </property>
       </widget>
       <widget class="QDial" name="lightMin">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>161</width>
          <height>161</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="lightName">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>700</y>
          <width>201</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Monaco</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Monaco&quot;;color: rgb(123, 22, 255);</string>
        </property>
        <property name="text">
         <string>light name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="backBuuton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>700</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Monaco</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Monaco&quot;;
color: rgb(123, 22, 255);</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QDial" name="lightValue">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>161</width>
          <height>161</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QDial" name="lightStep">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>520</y>
          <width>161</width>
          <height>171</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QDial" name="lightMax">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>350</y>
          <width>161</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Apple Chancery</family>
          <pointsize>14</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLCDNumber" name="stepValue">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>530</y>
          <width>201</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(123, 22, 255);</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="minValue">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>180</y>
          <width>201</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(123, 22, 255);</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="maxValue">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>350</y>
          <width>201</width>
          <height>151</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(123, 22, 255);</string>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>16</width>
          <height>741</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="cortex">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>391</width>
          <height>461</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Monaco&quot;;
color: rgb(123, 22, 255);</string>
        </property>
        <property name="title">
         <string>Temperature</string>
        </property>
        <widget class="QLCDNumber" name="temp1">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>30</y>
           <width>151</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(123, 22, 255);
</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="temp2">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>140</y>
           <width>151</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(123, 22, 255);</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="temp3">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>250</y>
           <width>151</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(123, 22, 255);</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="temp4">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>360</y>
           <width>151</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(123, 22, 255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>111</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Cortex 1</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>140</y>
           <width>111</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Cortex 2</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>260</y>
           <width>111</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Cortex 3</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>370</y>
           <width>111</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Cortex 4</string>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>10</y>
           <width>20</width>
           <height>441</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="back">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>590</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Monaco&quot;;
color: rgb(123, 22, 255);</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
